adv-java/src/main/java/dataModel/Delete.java

```diff
@@ -0,0 +1,27 @@
+package dataModel;
+import sqlDao.DeleteServlet; 
+import java.io.IOException;
+import javax.servlet.http.*;
+
+public class Delete extends HttpServlet{
```

Comment:
A bit more descriptive name for this servlet will help. The class/interface name should be able to convey what the purpose of the them is. Meaningful names are preferred.

Reply:
sure sir i will change the names